scripts/checkseverity.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Run this script to validate all managed-notifications are configured with a severity
+for file in $(find . -type f -name "*.json"); do
+    if ! jq .severity -e < $file > /dev/null 2>&1; then
```

Comment:
probably should double-quote this `$file` in case another script has a space in it.